ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsNoJobTask.java

```diff
@@ -85,7 +85,7 @@
  **/
 public class BasicStatsNoJobTask implements IStatsProcessor {
 
-  private static transient final Logger LOG = LoggerFactory.getLogger(BasicStatsNoJobTask.class);
+  private static final Logger LOG = LoggerFactory.getLogger(BasicStatsNoJobTask.class);
```

Comment:
good catch, I guess this rest be handled in a separate newbie patch: https://issues.apache.org/jira/browse/HIVE-28796

Reply:
reverted from this PR